src/test/java/net/sf/jabref/logic/formatter/FormatterTest.java

```diff
@@ -85,6 +85,8 @@ public FormatterTest(Formatter formatter) {
                 new Object[]{new SuperscriptFormatter()},
                 new Object[]{new UnitFormatter()},
                 new Object[]{new RemoveBracesFormatter()}
+                // The EraseFormatter is special since it always returns null to indicate that the field should be deleted.
+                // new Object[]{new EraseFormatter()}
```

Comment:
I do not like the EraseFormatter. I think this should not be seen as a formatter as it clearly violates the interface.